app/src/components/market/profile/redeem.tsx

```diff
@@ -116,14 +116,18 @@ export const Redeem = (props: Props) => {
     }
   }
 
+  const haveShares = balance.every((balanceItem: BalanceItems) => balanceItem.shares.isZero())
```

Comment:
This should only check if the user has shares for the winning token. If they have shares but only for the losing outcome, the redeem button should be disabled anyways.